Firmware/mmu2_reporting.cpp

```diff
@@ -289,6 +289,20 @@ void ReportProgressHook(CommandInProgress cip, uint16_t ec) {
     }
 }
 
+void TryLoadUnloadProgressbarInit() {
+    // Clear the status line
+    lcd_set_cursor(0, 3);
+    lcd_space(LCD_WIDTH);
+
+    // Reset cursor position
+    lcd_set_cursor(0, 3);
```

Comment:
Are you sure no other drawing function accesses the LCD while doing the progress bar?
Other functions may screw the cursor and we may start printing to some other position.

Reply:
That's a good point. I'll add back `lcd_putc_at`, it will increase code size probably but it should be safer for now.

Reply:
Fixed with [78ea5ee](https://github.com/prusa3d/Prusa-Firmware/pull/4076/commits/78ea5ee5da5b330877e0c17beff5c52b0df77a68)